Project Library - Kasia, Nella & Cholpon

cholpoun opened the pull request:
## Netlify link
https://movielibraryknc.netlify.app/

## Collaborators
[nella-x, kasssiii]

JennieDalgren reviewed (changes requested):
Great job with this project!

I'm not sure if it is a mistake or what it is, but you have two versions of html, css, js files. Make sure to only keep the ones you use.

You have managed well with creating functions for sorting and filtering. Also nice to see that you've tried out the switch statement. 

I'm not sure if it is because you have worked in team, but there is some inconsistency in your way of creating functions. either stick to function declaration with the keyword function, or stick to arrow functions. Don't mix this. Also, clean up your code from unused out-commented code. Or if you want to keep it, put a comment to why it's not "active" and why you still have it in your file.

Also in your css file you are mixing both rem and px values. Try to stick to one apporach, at least for the same properties such as font-size for example.

Clean up the code, at least the javascript one and remove the files that you don't need in your project to make this project approved.

Functionality wise, really good job with this project. And good work with the styling too!

smily342 commented:
Hey this is : Sherry , Hélène and me commenting.

Congratulation on your project again!

So first : 

We went trough tall your files, and  it was very easy to read, understandable and clean. 
Just noticed that you had like 2 "head" tag and was not perfectly close?

Overall, very good job !

cholpoun commented:
Hi @JennieDalgren 

Nella (nella-x), Kasia (kasssiii), and I have updated the files. 

- JS - updated functions and made them as an arrow functions
- CSS - updated rem to px 
- HTML - deleted unused code 
- Deleted extra CSS, JS and HTML files 

Could you please approve this PR? 

Thanks!

JennieDalgren reviewed (approved):
🎯
